content/admin/company/owners.md

```diff
@@ -15,14 +15,16 @@ As a company owner, you can configure [Single Sign-on (SSO)](../../security/for-
 
 1. Sign in to [Docker Hub](https://hub.docker.com/), navigate to the **Organizations** page, and select your company.
 2. In the **Overview** tab, select **Add owner** and enter their Docker ID or email address.
-3. Once you've selected **Add Owner**, you can view the user in the **Company Owners** table.
+3. After you find the user, select **Add company owner**.
 
-    ![company-overview](../images/company-overview.png)
+Once you've added the new owner, you can view the user in the **Company owners** table.
+
+![company-overview](../images/company-overview.png)
```

Comment:
**Note to reviewer:** I'm thinking of removing these screenshots since it's pretty straightforward. What do you think?

Reply:
Totally agree. Viewing a user in a table is easy to understand. I could see keeping the other image with the Action icon, unless there's some inline icon or some other way to show what it looks like.

Reply:
Agree, I kept the second image and removed the one of the table.